distribution/docs/src/main/jdocs/content/_transformers/mime-type-mapper-intro-contents.adoc

```diff
@@ -0,0 +1,20 @@
+:title: Mime Type Mapper Intro
+:type: transformerIntro
+:status: published
+:link: _mime_type_mapper_intro
+:summary: Resolves file extensions to mime types and vice versa.
+:input:
+:metacard:
```

Comment:
It does cause problems if the property doesn't exist, even if it's not set. 

That said, I'm going to simplify this by defining the types differently, so this won't be necessary.